src/main.rs

```diff
@@ -6,26 +6,44 @@ mod models;
 mod rest;
 
 use crate::rest::RestServer;
-use std::env::args;
+use clap::Parser;
 use std::io::{Write, stdout};
 use std::process::exit;
 
+#[derive(Parser, Debug)]
+#[command(about = "rocd server")]
+struct CLI {
+    /// server host
+    #[arg(long, default_value_t = String::from("127.0.0.1"))]
+    host: String,
+
+    /// server port
+    #[arg(long, default_value_t = 3000)]
```

Comment:
I suggest to add value_name=... for all options, it makes help more readable (e.g. PORT for this one)

Reply:
what do you mean?
```
$ cargo run -- -h
   Compiling rocd v0.1.0 (/home/paq/work/roc/rocd)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 2.65s
     Running `target/debug/rocd -h`
rocd server

Usage: rocd [OPTIONS]

Options:
      --host <HOST>   server host [default: 127.0.0.1]
      --port <PORT>   server port [default: 3000]
      --dump-openapi  dump OpenAPI specification in JSON format to stdout and exit
  -h, --help          Print hel
```

Reply:
Oops, I thought it had different default